examples/client/Locomotion/src/context/newRideContext/index.tsx

```diff
@@ -370,6 +374,7 @@ const RidePageContextProvider = ({ children }: {
       const formattedEstimations = formatEstimations(services, estimations, tags);
       setChosenService(ride.scheduledTo ? formattedEstimations.find((e: any) => e.currency)
         : formattedEstimations.find((e: any) => e.eta));
+      setDefaultService(formattedEstimations?.[0]);
```

Comment:
default service should be the same as chosen service

Reply:
this is a case where are no available services, but we still want to use a service for a future ride. so there is no chosen service